README.md

```diff
@@ -59,26 +59,16 @@ chitchat = Route(
 
 # we place both of our decisions together into single list
 routes = [politics, chitchat]
-```
-
-We have our routes ready, now we initialize an embedding / encoder model. We currently support a `CohereEncoder` and `OpenAIEncoder` — more encoders will be added soon. To initialize them we do:
-
-```python
-import os
-from semantic_router.encoders import CohereEncoder, OpenAIEncoder
 
-# for Cohere
-os.environ["COHERE_API_KEY"] = "<YOUR_API_KEY>"
-encoder = CohereEncoder()
+# Example: Using a local encoder (no API key required)
+from semantic_router.encoders import LocalEncoder
+encoder = LocalEncoder()  # Uses 'BAAI/bge-small-en-v1.5' by default
 
-# or for OpenAI
-os.environ["OPENAI_API_KEY"] = "<YOUR_API_KEY>"
-encoder = OpenAIEncoder()
-```
+# If you want to use a different model:
+# encoder = LocalEncoder(name="BAAI/bge-base-en-v1.5")
 
-With our `routes` and `encoder` defined we now create a `RouteLayer`. The route layer handles our semantic decision making.
+# With our `routes` and `encoder` defined we now create a `RouteLayer`. The route layer handles our semantic decision making.
 
```

Comment:
no need for us to change this - also not sure why `#` was added to line 70 - that makes the entire sentence a H1 header